tests/Functional/ContentProjectRenderTest.php

```diff
@@ -196,40 +201,34 @@ public function testProjectRendering(): void {
-    $this->assertEquals('Start date', $labels[1]->getText());
-    $values = $description_lists[0]->findAll('css', 'dd.ecl-description-list__definition');
-    $this->assertEquals('10.05.2020', $values[1]->getText());
+    $first_field_list_expected_values['items'][1] = [
+      'label' => 'Start date',
+      'body' => '10.05.2020',
+    ];
+    $field_list_assert->assertPattern($first_field_list_expected_values, $description_lists[0]->getHtml());
 
     // Assert the second description list block's labels and values.
-    $values = $description_lists[1]->findAll('css', 'dd.ecl-description-list__definition');
-    $definition_element = $values[1];
-    $values = $definition_element->findAll('css', 'div');
-    $this->assertEquals('<div>€50</div>', trim($values[0]->getOuterHtml()));
-    $this->assertEquals('<div class="ecl-u-mt-m">50% of the overall budget</div>', trim($values[1]->getOuterHtml()));
+    $second_field_list_expected_values['items'][1]['body'] = "€5050% of the overall budget";
+    $field_list_assert->assertPattern($second_field_list_expected_values, $description_lists[1]->getHtml());
 
     // Assert the third description list block's labels and values.
-    $labels = $description_lists[2]->findAll('css', 'dt.ecl-description-list__term');
-    $this->assertCount(1, $labels);
-    $this->assertEquals('Project website', $labels[0]->getText());
-    $values = $description_lists[2]->findAll('css', 'dd.ecl-description-list__definition');
-    $this->assertCount(1, $values);
-    $values[0]->hasLink('Example website');
+    $third_field_list_expected_values = [
+      'items' => [
+        [
+          'label' => 'Project website',
+          'body' => 'Example website',
+        ],
+      ],
+    ];
+    $field_list_html = $description_lists[2]->getHtml();
+    $field_list_assert->assertPattern($third_field_list_expected_values, $field_list_html);
+    $field_list_assert->assertVariant('featured_horizontal', $field_list_html);
 
     // Assert documents file.
     $file_wrapper = $this->assertSession()->elementExists('css', 'div#project-documents');
-    $file_row = $file_wrapper->find('css', '.ecl-file .ecl-file__container');
-    $file_title = $file_row->find('css', '.ecl-file__title');
-    $this->assertContains('Test document project_document', $file_title->getText());
-    $file_info_language = $file_row->find('css', '.ecl-file__info div.ecl-file__language');
-    $this->assertContains('English', $file_info_language->getText());
-    $file_info_properties = $file_row->find('css', '.ecl-file__info div.ecl-file__meta');
-    $this->assertContains('(2.96 KB - PDF)', $file_info_properties->getText());
-    $file_download_link = $file_row->find('css', '.ecl-file__download');
-    $this->assertContains('/sample_project_document.pdf', $file_download_link->getAttribute('href'));
-    $this->assertContains('Download', $file_download_link->getText());
+    $this->assertMediaDocumentDefaultRender($file_wrapper, 'project_document');
```

Comment:
We shouldn't hardcode assertion values below in the assertion method `assertMediaDocumentDefaultRender(...)`, let's instead pass them as method arguments, so they can be set by the caller.

```
    $this->assertContains('English', $file_info_language->getText());
    $this->assertContains('(2.96 KB - PDF)', $file_info_properties->getText());
```